examples/src/main/scala/example/Http4sExample.scala

```diff
@@ -77,19 +80,31 @@ object Http4sExample extends IOApp with Common {
   // Our main app resource
   def server[F[_]: Async: Network: TracerProvider: Tracer: MeterProvider]: Resource[F, Server] =
     for {
-      clientMiddleware <- ClientMiddlewareBuilder.default(redactor).build.toResource
+      clientMiddleware <- ClientMiddlewareBuilder(
+        ClientSpanDataProvider
+          .openTelemetry(redactor)
+          .withUrlTemplateClassifier(urlTemplateClassifier)
+          .optIntoHttpRequestHeaders(HeaderRedactor.default)
+          .optIntoHttpResponseHeaders(HeaderRedactor.default)
+          .optIntoUrlScheme
```

Comment:
👍

Reply:
thanks! it's also required by the spec >.>